content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -26,3 +51,36 @@ version. [Learn more about availability during upgrades here](../system-overview
     "Check Now" button; the new version should be recognized.
 1. Click "View Update".
 1. Review the release notes and then click "Install Update".
+
+### Replicated Command Line Interface
```

Comment:
You should add a step here to tell the reader to upload the desired airgap package to the `AirgapPackagePath`.